packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -41,3 +42,10 @@ export class DataObjectWithCounter extends DataObject {
         this.isRunning = false;
     }
 }
+
+export const dataObjectWithCounterFactory = new DataObjectFactory(
+    DataObjectWithCounter.type,
+    DataObjectWithCounter,
+    allFactories,
```

Comment:
This only creates a SharedCounter so it should only have its factory in the registry entry.